pages/api/FolderApi.tsx

```diff
@@ -0,0 +1,17 @@
+import getDate from "./GetData";
+
+export async function getFolderUserData() {
```

Comment:
내용으로 보면 이런 시그니쳐가 함수를 더 잘 설명해줄거같아요
```suggestion
export async function getUserData(userId = DEFAULT_USER_ID) {
```